visual-python/src/saucelabs_visual/client.py

```diff
@@ -0,0 +1,185 @@
+from os import environ
+from typing import List
+
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport, BasicAuth
+
+from saucelabs_visual.regions import Region
+from saucelabs_visual.typing import IgnoreRegion, FullPageConfig
+
+
+class SauceLabsVisual:
+    client: Client = None
+    build_id: str = None
+
+    def __init__(self):
+        self._create_client()
+
+    def _create_client(self):
+        region_url = Region.from_name(environ.get("SAUCE_REGION") or 'us-west-1').graphql_endpoint
+        username = environ.get("SAUCE_USERNAME")
+        access_key = environ.get("SAUCE_ACCESS_KEY")
+
+        if username is None or access_key is None:
+            raise Exception(
+                'Sauce Labs credentials not set. Please check that you set correctly your `user` '
+                'and `key`.'
+            )
+
+        transport = AIOHTTPTransport(
+            url=region_url,
+            auth=BasicAuth(environ.get("SAUCE_USERNAME"), environ.get("SAUCE_ACCESS_KEY")),
```

Comment:
you already have username and access key - line 20 & 21
```suggestion
            auth=BasicAuth(username, access_key),
```

Reply:
Yep totally forgot to replace those after refactoring it. Thanks!